L/libcint/build_tarballs.jl

```diff
@@ -39,7 +41,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="OpenBLAS32_jll", uuid="656ef2d0-ae68-5445-9ca0-591084a874a2"))
+    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363"))
```

Comment:
Unless you rename all BLAS symbols, I'm not sure you get much benefit from this change

Reply:
Thanks for the reminder! I thought they use the same symbol names except one has a newer version. I've changed the dependency back to `OpenBLAS32_jll`.